recipe/meta.yaml

```diff
@@ -179,9 +179,10 @@ outputs:
         - cmake
         - m2-make   # [win]
         - posix     # [win]
+        # hack (around which bug?)
         - llvm-openmp   # [linux and blas_impl == "openblas"]
       host:
-      # Building with blis fails due to a conda-build bug
+      # Building with blis fails due to a conda-build bug - which one?
```

Comment:
Same here - which conda-build bug is relevant here? What would be necessary to simply use:
```
      host:
        - {{ pin_subpackage("libblas", exact=True) }}
        - {{ pin_subpackage("libcblas", exact=True) }}
      {% if blas_impl != 'blis' %}
        - {{ pin_subpackage("liblapack", exact=True) }}
        - {{ pin_subpackage("liblapacke", exact=True) }}
      {% endif %}
```